eox-core-v-1-0/prose/edit/src/design-considerations-01-construction-principles.md

```diff
@@ -1,10 +1,32 @@
 ## Construction Principles
 
-TODO:
+An End-of-X (EOX) lifecycle statement represented by an OpenEoX Core document is created through the application of
+well-defined schemas during a process of collaboration between multiple stakeholders including vendors,
+Product Security Incident Response Teams (PSIRTs), and lifecycle management authorities.
+The focused and purposefully minimal schema is designed to be orchestrated with product identification schemas and
+put into context serving the goal of providing actionable, structured, and validated lifecycle information targeting a high degree of automation.
 
-- add generic paragraph about OpenEoX Core
-- add paragraph about JSON Schema
-- optional: add paragraph about imported/references schemas (should be none here)
+The format chosen is JSON Schema Draft 2020-12 which allows validation and delegation to sub-schema providers.
+The latter aligns well with separation of concerns and enables the OpenEoX Core schema to be imported and referenced
+by other standards, specifications, and product-specific implementations.
+
+Technically, the use of JSON Schema Draft 2020-12 allows validation and proof of model conformance through
+established schema-based validation of the declared lifecycle information inside OpenEoX Core documents.
```

Comment:
See above - please use OpenEoX Core information or data.

Reply:
Thank you! Suggestion applied and corrected!